packages/ui/src/components/Lessons/ManageLesson/ManageLessonDialog.tsx

```diff
@@ -147,6 +154,9 @@ export const ManageLessonDialog: React.FC<{
   duration?: number;
   slots: IAvailabilitySlot.Self[];
   bookedSlots: IAvailabilitySlot.SubSlot[];
+  isVerified: boolean;
+  hasBookedLessons: boolean;
```

Comment:
To keep it consistent.

```suggestion
  hasBookedLesson: boolean;
```